experimenter/tests/integration/nimbus/android/adbrun.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import os
+from pathlib import Path
+
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+
+class ADBrun:
+    binary = "adbrun"
+    logger = logging.getLogger()
+    here = Path()
+
+    def launch(self):
+        # First close sim if any then launch
+        os.system(
+            "~/Library/Android/sdk/platform-tools/adb devices | grep emulator | cut -f1 | while read line; do ~/Library/Android/sdk/platform-tools/adb -s $line emu kill; done"  #  NOQA
+        )
+        # Then launch sim
+        os.system("sh launchSimScript.sh")
```

Comment:
This should probably be a bash script that we invoke with subprocess.check_result. Remember to `set -euo pipefail`

Reply:
This is actually not being used so ill delete it. More of a carry-over from the mozilla central workflow.